### PR TITLE
Add price filter hook for functionality

### PR DESCRIPTION
**Pull request by DLi7077**
Derived A custom hook (usePriceFilter.js) from PriceFilter that returns the filter component and the matching price filter
Created another custom hook (useFilters.js) to store all search filters.
- We will add food categories for this hook when completed

Link with search page to filter restaurants

### CODE REVIEW

**Review by Remos96 — DISMISSED** [review]
LG

File: client/src/hooks/filters/index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import usePriceFilter from "./usePriceFilter";
+
+export default function useFilters() {
+  const { matchesPriceCategory, PriceFilterComponent } = usePriceFilter();
```
Comment (Remos96) [inline client/src/hooks/filters/index.js:5]: I think I have a rough idea of what's going on here, could you clarify exactly what's happening exactly?

Comment (DLi7077) [reply client/src/hooks/filters/index.js:5]: I made a custom hook off that returns a function and a component.

The function is the Boolean function that checks if a restaurant matches the current price category

The component is manav's price filter view


Similar to how useState returns a variable and a function to modify the variable,
I created a hook that returns
1. The matching condition
2. The component to render

Comment (Remos96) [reply client/src/hooks/filters/index.js:5]: Ah, thank you!

**Review by Remos96 — APPROVED** [review]

**Pull request merged**